api/tests/python/nro_services/test_nro_request_utils.py

```diff
@@ -423,97 +422,97 @@ def test_update_nr_header_with_mutating_priority(freeze_datetime_utcnow, test_na
      ],
      State.REJECTED
      ),
     ('COMPLETED',
-     [{'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C'}],
+     [{'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'}],
      State.CONDITIONAL
      ),
     ('COMPLETED',
      [
-         {'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'NE'},
-         {'choice_number': 2, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C'},
+         {'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
+         {'choice_number': 2, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
      ],
      State.CONDITIONAL
      ),
     ('COMPLETED',
      [
-         {'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'A'},
-         {'choice_number': 2, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C'},
+         {'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'A', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
+         {'choice_number': 2, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
      ],
      State.CONDITIONAL
      ),
     ('COMPLETED',
      [
-         {'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'R'},
-         {'choice_number': 2, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C'},
+         {'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'R', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
+         {'choice_number': 2, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
      ],
      State.CONDITIONAL
      ),
     ('COMPLETED',
      [
-         {'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C'},
-         {'choice_number': 2, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'A'},
+         {'choice_number': 1, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'C', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
+         {'choice_number': 2, 'name': 'PROCINE ENTERPRISES LTD', 'designation': None, 'name_state_type_cd': 'A', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
      ],
```

Comment:
Again, only the approved names make sense with a consumption date. The others should be blank.